src/near-nativePayment.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  near,
+  log,
+  json,
+  BigInt,
+  Bytes,
+  crypto,
+  ByteArray,
+  JSONValueKind,
+  BigDecimal,
+  typeConversion,
+} from "@graphprotocol/graph-ts";
+import { NearPayment } from "../generated/near/schema";
+
+export function handleReceipt(receipt: near.ReceiptWithOutcome): void {
+  const actions = receipt.receipt.actions;
+  for (let i = 0; i < actions.length; i++) {
+    handleAction(actions[i], receipt);
+  }
+}
+
+function generateId(transactionHash: Bytes): string {
+  var id = transactionHash.toHex(); // + paymentReference.toHex().slice(2);
+  return crypto.keccak256(ByteArray.fromHexString(id)).toHex();
+}
```

Comment:
if there is only one value, I don't think this is necessary, you can use the receiptId as the ID.

To consider for the ID:
- a transaction might contain several transfers
- a single payment reference might be present in several transactions